### PR TITLE
Generate constant cpu load for a configured duration

### PR DESCRIPTION
**Pull request by pradykaushik**

### CODE REVIEW

**Comment by pradykaushik** [comment]
When merging, make sure to change version to 2.1

File: src/main/java/loadgenerator/strategies/ConstantCPULoad.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        } catch (Exception e) {
+            System.err.println("failed to obtain processor architecture information.");
+            e.printStackTrace();
+        }
+        // Printing the processor architecture information
+        System.out.println(processorArchInfo);
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class Builder {
```
Comment (bm1216) [inline src/main/java/loadgenerator/strategies/ConstantCPULoad.java:60]: Where do you actually set the values obtained from the config file to the variables in the Builder class?

**Review by bm1216 — CHANGES_REQUESTED** [review]
Just wanted to ask if the changes you're making actually makes use of the config file provided, and if so, where

File: src/main/java/loadgenerator/strategies/ConstantCPULoad.java (hunk 1 above)
Comment (pradykaushik) [reply src/main/java/loadgenerator/strategies/ConstantCPULoad.java:60]: The yaml config file is deserialized into the Builder instance using Jackson.

**Comment by pradykaushik** [comment]
@bm1216 thank you for taking the initiative of reviewing the PR!!.
I have pushed a couple more changes.

**Pull request merged**